packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -437,19 +437,19 @@ curl -v -X POST \
 
 <ApiOperation method="delete" url="/api/v1/inlineHooks/${id}" />
 
-##### Request Parameters
+##### Request parameters
 
 | Parameter | Description                            | Param Type   | DataType   | Required |
 | --------- | -------------------------------------- | ------------ | ---------- | -------- |
 | `id`      | The ID of the Inline Hook to delete.   | Path         | String     | TRUE     |
 
 Deletes the Inline Hook matching the provided `id`. Once deleted, the Inline Hook is unrecoverable. As a safety precaution, only Inline Hooks with a status of `INACTIVE` are eligible for deletion.
```

Comment:
Deletes the Inline Hook that matches the provided `id`. After it is deleted, the Inline Hook is unrecoverable. As a safety precaution, only Inline Hooks with a status of `INACTIVE` are eligible for deletion.